torch/csrc/jit/register_c10_ops.cpp

```diff
@@ -23,12 +23,12 @@ IValue unwrap(IValue&& ivalue) {
   if (ivalue.isTensor() && ivalue.toTensor().defined()) {
     return unwrap_tensor(std::move(ivalue).toTensor());
   } else if (ivalue.isTensorList()) {
-    for (auto& item : ivalue.toTensorList()->elements()) {
+    for (at::Tensor item : ivalue.toTensorList()->elements()) {
```

Comment:
This looks wrong. Assignment on the line below was doing something, this no longer does it.

Reply:
good catch, thanks.